### DISCUSSION TITLE
undefined reading showPopup

### DISCUSSION THREAD

**Discussion by ArunPatal** [ajaxorg/ace#5330] [General]
Hi,

Am getting following error while displaying completer.

![bug](https://github.com/ajaxorg/ace/assets/30784805/88ba49b8-980c-4535-961d-559aa3aaa753)

Below are my settings. 

```
editor.setOptions({
  enableBasicAutocompletion: false, //boolean: true to show local
  enableSnippets: false, //boolean: true to show snippets
  enableLiveAutocompletion: false, //
});

editor.execCommand("startAutocomplete");

if(editor.completers == undefined){
       editor.completers = [];
};

editor.completers[0] = My_Completer;
editor.completer.showPopup(editor);
```

**Comment by nightwing** [comment]
if you disable the option `enableBasicAutocompletion`  `editor.completer` is not defined

**Reply by ArunPatal** [reply]
But what if don't want basic autocompletion and only the self defined one?
